### ISSUE TITLE
[bug]: Cannot import certain models, Invoke process exited with code 3221225477

### ISSUE THREAD

**Issue by Guilded-Aurora** [invoke-ai/InvokeAI#9245]
### Is there an existing issue for this problem?

- [x] I have searched the existing issues

### Install method

Invoke's Launcher

### Operating system

Windows

### GPU vendor

Nvidia (CUDA)

### GPU model

RTX 1060

### GPU VRAM

6 GB

### Version number

v1.8.1 Launcher/v6.13.0

### Browser

N/A

### System Information

{
    "version": "6.13.0",
    "dependencies": {
        "absl-py"              : "2.4.0",
        "accelerate"           : "1.13.0",
        "annotated-types"      : "0.7.0",
        "anyio"                : "4.13.0",
        "attrs"                : "26.1.0",
        "bcrypt"               : "3.2.2",
        "bidict"               : "0.23.1",
        "bitsandbytes"         : "0.49.2",
        "blake3"               : "1.0.8",
        "certifi"              : "2022.12.7",
        "cffi"                 : "2.0.0",
        "charset-normalizer"   : "2.1.1",
        "click"                : "8.4.1",
        "colorama"             : "0.4.6",
        "coloredlogs"          : "15.0.1",
        "compel"               : "2.1.1",
        "contourpy"            : "1.3.3",
        "cryptography"         : "48.0.0",
        "CUDA"                 : "12.8",
        "cycler"               : "0.12.1",
        "Deprecated"           : "1.3.1",
        "diffusers"            : "0.37.0",
        "dnspython"            : "2.8.0",
        "dynamicprompts"       : "0.31.0",
        "ecdsa"                : "0.19.2",
        "einops"               : "0.8.2",
        "email-validator"      : "2.3.0",
        "fastapi"              : "0.118.3",
        "fastapi-events"       : "0.12.2",
        "filelock"             : "3.29.0",
        "flatbuffers"          : "25.12.19",
        "fonttools"            : "4.63.0",
        "fsspec"               : "2026.4.0",
        "gguf"                 : "0.19.0",
        "h11"                  : "0.16.0",
        "httpcore"             : "1.0.9",
        "httptools"            : "0.8.0",
        "httpx"                : "0.28.1",
        "huggingface_hub"      : "0.36.2",
        "humanfriendly"        : "10.0",
        "idna"                 : "3.4",
        "importlib_metadata"   : "7.1.0",
        "InvokeAI"             : "6.13.0",
        "jax"                  : "0.7.1",
        "jaxlib"               : "0.7.1",
        "Jinja2"               : "3.1.6",
        "kiwisolver"           : "1.5.0",
        "MarkupSafe"           : "3.0.3",
        "matplotlib"           : "3.10.9",
        "mediapipe"            : "0.10.14",
        "ml_dtypes"            : "0.5.4",
        "mpmath"               : "1.3.0",
        "networkx"             : "3.6.1",
        "numpy"                : "1.26.4",
        "onnx"                 : "1.16.1",
        "onnxruntime"          : "1.19.2",
        "opencv-contrib-python": "4.11.0.86",
        "opt_einsum"           : "3.4.0",
        "packaging"            : "24.1",
        "passlib"              : "1.7.4",
        "picklescan"           : "1.0.4",
        "pillow"               : "12.2.0",
        "prompt_toolkit"       : "3.0.52",
        "protobuf"             : "4.25.9",
        "psutil"               : "7.2.2",
        "pyasn1"               : "0.6.3",
        "pycparser"            : "3.0",
        "pydantic"             : "2.13.4",
        "pydantic-settings"    : "2.14.1",
        "pydantic_core"        : "2.46.4",
        "pyparsing"            : "3.3.2",
        "PyPatchMatch"         : "1.0.2",
        "pyreadline3"          : "3.5.6",
        "python-dateutil"      : "2.9.0.post0",
        "python-dotenv"        : "1.2.2",
        "python-engineio"      : "4.13.2",
        "python-jose"          : "3.5.0",
        "python-multipart"     : "0.0.29",
        "python-socketio"      : "5.16.2",
        "PyWavelets"           : "1.9.0",
        "PyYAML"               : "6.0.3",
        "regex"                : "2026.5.9",
        "requests"             : "2.28.1",
        "rsa"                  : "4.9.1",
        "safetensors"          : "0.7.0",
        "scipy"                : "1.17.1",
        "semver"               : "3.0.4",
        "sentencepiece"        : "0.2.0",
        "setuptools"           : "70.2.0",
        "simple-websocket"     : "1.1.0",
        "six"                  : "1.17.0",
        "sounddevice"          : "0.5.5",
        "spandrel"             : "0.4.2",
        "starlette"            : "0.48.0",
        "sympy"                : "1.14.0",
        "tokenizers"           : "0.22.2",
        "torch"                : "2.7.1+cu128",
        "torchsde"             : "0.2.6",
        "torchvision"          : "0.22.1+cu128",
        "tqdm"                 : "4.66.5",
        "trampoline"           : "0.1.2",
        "transformers"         : "4.57.6",
        "typing-inspection"    : "0.4.2",
        "typing_extensions"    : "4.15.0",
        "urllib3"              : "1.26.13",
        "uvicorn"              : "0.48.0",
        "watchfiles"           : "1.2.0",
        "wcwidth"              : "0.7.0",
        "websockets"           : "16.0",
        "wrapt"                : "2.2.1",
        "wsproto"              : "1.3.2",
        "xformers"             : "0.0.31.post1",
        "zipp"                 : "3.19.2"
    },
    "config": {
        "schema_version": "4.0.3",
        "legacy_models_yaml_path": null,
        "host": "127.0.0.1",
        "port": 9090,
        "allow_origins": [],
        "allow_credentials": true,
        "allow_methods": ["*"],
        "allow_headers": ["*"],
        "ssl_certfile": null,
        "ssl_keyfile": null,
        "log_tokenization": false,
        "patchmatch": true,
        "models_dir": "models",
        "convert_cache_dir": "models\\.convert_cache",
        "download_cache_dir": "models\\.download_cache",
        "legacy_conf_dir": "configs",
        "db_dir": "databases",
        "outputs_dir": "outputs",
        "image_subfolder_strategy": "flat",
        "custom_nodes_dir": "nodes",
        "style_presets_dir": "style_presets",
        "workflow_thumbnails_dir": "workflow_thumbnails",
        "log_handlers": ["console"],
        "log_format": "color",
        "log_level": "info",
        "log_sql": false,
        "log_level_network": "warning",
        "use_memory_db": false,
        "dev_reload": false,
        "profile_graphs": false,
        "profile_prefix": null,
        "profiles_dir": "profiles",
        "max_cache_ram_gb": null,
        "max_cache_vram_gb": null,
        "log_memory_usage": false,
        "model_cache_keep_alive_min": 0,
        "device_working_mem_gb": 3,
        "enable_partial_loading": true,
        "keep_ram_copy_of_weights": true,
        "ram": null,
        "vram": null,
        "lazy_offload": true,
        "pytorch_cuda_alloc_conf": null,
        "device": "auto",
        "precision": "auto",
        "sequential_guidance": false,
        "attention_type": "auto",
        "attention_slice_size": "auto",
        "force_tiled_decode": false,
        "pil_compress_level": 1,
        "max_queue_size": 10000,
        "clear_queue_on_startup": false,
        "max_queue_history": null,
        "allow_nodes": null,
        "deny_nodes": null,
        "node_cache_size": 512,
        "hashing_algorithm": "blake3_single",
        "remote_api_tokens": null,
        "scan_models_on_startup": false,
        "unsafe_disable_picklescan": false,
        "allow_unknown_models": true,
        "multiuser": false,
        "strict_password_checking": false,
        "external_alibabacloud_api_key": null,
        "external_alibabacloud_base_url": null,
        "external_gemini_api_key": null,
        "external_openai_api_key": null,
        "external_gemini_base_url": null,
        "external_openai_base_url": null,
        "external_seedream_api_key": null,
        "external_seedream_base_url": null
    },
    "set_config_fields": ["legacy_models_yaml_path"]
}

### What happened

I attempted to import a model from my previous existing models in ComfyUI, whenever the hash step completes, the python back-end crashes and after 10-20 seconds gives an exit code in the launcher console.

```
H:\[Redacted]\[Invoke_Dir]\.venv\Lib\site-packages\transformers\utils\hub.py:110: FutureWarning: Using `TRANSFORMERS_CACHE` is d
eprecated and will be removed in v5 of Transformers. Use `HF_HOME` instead.
  warnings.warn(
[2026-05-██ ██:30:21,166]::[InvokeAI]::INFO --> Using torch device: NVIDIA GeForce GTX 1060
[2026-05-██ ██:30:22,387]::[InvokeAI]::INFO --> cuDNN version: 90701
[2026-05-██ ██:30:28,289]::[InvokeAI]::INFO --> Patchmatch initialized
[2026-05-██ ██:30:30,443]::[InvokeAI]::INFO --> InvokeAI version 6.13.0
[2026-05-██ ██:30:30,444]::[InvokeAI]::INFO --> Root directory = H:\[Redacted]\[Invoke_Dir]
[2026-05-██ ██:30:30,450]::[InvokeAI]::INFO --> Initializing database at H:\[Redacted]\[Invoke_Dir]\databases\invokeai.db
[2026-05-██ ██:30:30,496]::[InvokeAI]::INFO --> JWT secret loaded from database
[2026-05-██ ██:30:30,666]::[ModelManagerService]::INFO --> [MODEL CACHE] Calculated model RAM cache size: 4096.00 MB. Heuristics
 applied: [1, 2, 3].
[2026-05-██ ██:30:30,680]::[ModelInstallService]::INFO --> Restoring incomplete installs
[2026-05-██ ██:30:30,681]::[ModelInstallService]::INFO --> Finished restoring incomplete installs
[2026-05-██ ██:30:30,760]::[InvokeAI]::INFO --> Invoke running on http://127.0.0.1:9090 (Press CTRL+C to quit)
[2026-05-██ ██:31:11,108]::[InvokeAI]::INFO --> Started installation of H:\[Redacted]\[Comfy_Install]\ComfyUI\models\check
points\dreamshaperXL10_alpha2Xl10.safetensors
[2026-05-██ ██:31:11,111]::[ModelInstallService]::INFO --> Model install started: H:/[Redacted]/[Comfy_Install]/ComfyUI/mo
dels/checkpoints/dreamshaperXL10_alpha2Xl10.safetensors
Hashing dreamshaperXL10_alpha2Xl10.safetensors: 100%|██████████████████████████████████████████| 1/1 [00:23<00:00, 23.32s/file]
Invoke process exited with code 3221225477
```

Using this command, I created a dump of relevant error details:
```
Get-WinEvent -FilterHashtable @{LogName='Application'; ProviderName='Application Error'} -MaxEvents 5 |
Select-Object TimeCreated, Message | Format-List
```  

```
TimeCreated : 5/██/2026 ██:28:24 ██
Message     : Faulting application name: python.exe, version: 3.12.9150.1013, time stamp: [Redacted]
              Faulting module name: torch_cpu.dll, version: 0.0.0.0, time stamp: [Redacted]
              Exception code: 0xc0000005
              Fault offset: 0x0000000006013f94
              Faulting process id: [Redacted]
              Faulting application start time: [Redacted]
              Faulting application path: C:\Users\[Redacted]\AppData\Roaming\uv\data\python\cpython-3.12.9-windows-x86_64-none\python.exe
              Faulting module path: H:\[Redacted]\[Redacted]\.venv\Lib\site-packages\torch\lib\torch_cpu.dll
              Report Id: [Redacted]
              Faulting package full name:
              Faulting package-relative application ID:
```

Here is the link to the model that I used:
https://huggingface.co/nolual/majic/blob/main/dreamshaperXL10_alpha2Xl10.safetensors

From what this is telling me, torch is throwing some manner of error upon exiting the hashing step, however it should go without saying that this model has no problems importing and running in ComfyUI, and has never thrown me any manner of issues. This leads me to conclude some point following the conclusion of this step, it isn't properly handled by InvokeAI. Naturally, I am not remotely acquainted with this code base, so I couldn't really say for certain this isn't user error, i.e, installation on a separate drive somehow causing some manner of issue. 

I also did a control experiment, using a different model:
```
[ModelInstallService]::INFO --> Model install started: H:/[Redacted]/[Comfy_Install]/ComfyUI/mo
dels/checkpoints/anything-v4.5-inpainting.safetensors
Hashing anything-v4.5-inpainting.safetensors: 100%|████████████████████████████████████████████| 1/1 [00:12<00:00, 12.94s/file]
[ModelInstallService]::INFO --> Model install complete: H:/[Redacted]/[Comfy_Install]/ComfyUI/m
odels/checkpoints/anything-v4.5-inpainting.safetensors
```

It properly imports this model, so it isn't ALL imports that aren't working.

### What you expected to happen

...I expected the model to import.

### How to reproduce the problem

It does state in the log that this is an import, but I'll lay out a bit more deeply where in the menu I went. I opened the launcher, the launcher opened the interface, from the interface, I selected
Models -> Scan Folder
Then I put the location in, pressed 'Scan Folder', then manually selected the model I was trying to import, dreamshaperXL10. It then began the hashing step, upon waiting 10-20 seconds, it finishes the hashing step. At that point, 5-10 seconds later I got a 'python crashed' windows prompt, then 10-20 seconds later I get the exit code in the console.

Link to model:
https://huggingface.co/nolual/majic/blob/main/dreamshaperXL10_alpha2Xl10.safetensors

### Additional context

I attempted to search for pre-existing forms of this issue, but the only thing I saw related to downloading models, and specifically flux related issues. This model is an XL fine-tune, it's not something I could find any pre-existing bug reports or solutions for.

**Comment by Pfannkuchensack** [comment]
Do you have windows pagefile on auto? Could be that the hashing cant be completed because running out of ram.

you can change the hash algorithm to random if nothing works. https://invoke.ai/configuration/invokeai-yaml/#model-hashing

**Comment by Guilded-Aurora** [comment]
> Do you have windows pagefile on auto? Could be that the hashing cant be completed because running out of ram.
> 
> you can change the hash algorithm to random if nothing works. https://invoke.ai/configuration/invokeai-yaml/#model-hashing

I have just attempted what you suggested:
```
# Internal metadata - do not edit:
schema_version: 4.0.3

# Put user settings here - see https://invoke.ai/configuration/invokeai-yaml/:
hashing_algorithm: random
```

This is what I set in 'invokeai.yaml'. I then proceeded to restart the server through the launcher, and continued to get the exact same error. I've also restarted the launcher and proceeded to launch the server again just in case, same thing. I monitored RAM usage during this process and there were no spikes, which may indicate that it skipped, as it did seem to finish immediately. I'm fairly certain the hashing completed just fine before making this change, it's something that happens following it before any other console outputs that is the actual cause.

**Comment by alexwyattdev** [comment]
I've been experiencing a lot of issues like this lately. As the previous commenter said, RAM does not look like an issue. It feels absolutely random and does not matter what model I am trying to install.

**Comment by shanedk** [comment]
I'm getting this bug now with Krea 2 models, sometimes when installing, sometimes when invoking.
